src/services/ombd-service.js

```diff
@@ -0,0 +1,49 @@
+import 'es6-promise';
```

Comment:
Thank god there is a checkbox to hide all comments, right arround ......................... ^ here

Hound is not yer configured

Reply:
+1